### PR TITLE
updated the venue

### PR DESCRIPTION
**Pull request by dada4lsgs**
### what's updated
the place to take

### CODE REVIEW

**Comment by ealo4kiki** [comment]
no problem!

File: index.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h2>第２回Git勉強会</h2>
                     <article>
                         <h3>イベント日時・場所</h3>
                         <p>3月23日 19:00開始</p>
-                        <p>株式会社〇〇 イベントセミナー会場</p>
+                        <p>株式会社インプレス イベントセミナー会場</p>
```
Comment (ealo4kiki) [inline index.html]: add the number of  the place

**Review by ealo4kiki — CHANGES_REQUESTED** [review]
check my comment of line 58

**Comment by dada4lsgs** [comment]
I've changed

**Comment by ealo4kiki** [comment]
ok

**Review by ealo4kiki — APPROVED** [review]

**Pull request merged**